### ISSUE TITLE
Implement watch lists

### ISSUE THREAD

**Issue by jar398** [OpenTreeOfLife/opentree#551]
That is, a curator can elect to 'watch' a particular study (or collection...) which means they get email every time the study is changed (i.e. new commit).

Many implementation details to be worked out obviously.

**Comment by kcranston** [comment]
Is this different from the favourites functionality mentioned in issue #675 ?

**Comment by jar398** [comment]
If something is on a watch list, you get a notification when it changes.

Suppose you add a study, and then three months later someone makes an edit.
Just as with wikipedia, if you care about your curation effort, you care
when someone starts to muck with it, and checking it periodically is not a
workable approach.

This would be one line of defense against vandalism, but its interest goes
beyond that.

**Comment by kcranston** [comment]
I understand the functionality that you are describing. Just wondering if we are talking about the same feature with favourite / watch / star (i.e. we have an issue that requests `favourite` and one requesting `watch` and it seems unnecessary to have two different buttons for the same or very similar functionality).

**Comment by jimallman** [comment]
Yes, my mockups to date assume a single favorites feature that would handle both "bookmarking" collections (for easy checking later) and notification if desired.

**Comment by josephwb** [comment]
+1 In addition to notification, it would be nice to see a summary of what has changed. At the moment the "History" tab in the curator is not all that useful (especially if a poor commit message was used, or none at all). Maybe, in addition to including such information in a notification email, the "History" tab could link to the git diff itself. For example, @snacktavish sent me [this](https://github.com/OpenTreeOfLife/phylesystem-1/commit/5817ed1ecc98b762652fcd68fbed9eceb0773ba3) so that I could see what she did.

**Comment by jimallman** [comment]
> Maybe, in addition to including such information in a notification email, the "History" tab could link to the git diff itself. For example, @snacktavish sent me this so that I could see what she did.

Ah, great minds think alike. We can certainly build these diff links and add them to the History tab.
